### PR TITLE
Add missing keywords of Agility

### PR DESCRIPTION
**Pull request by x09326**
PR for #8073 
@pvnr0082t please help me review it, thanks.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/arturadib/agility
 * Official website (optional, not the repository):http://agilityjs.com
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars:558
   - Count of watchers:33
   - Count of forks:77
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

cc #8073

### CODE REVIEW

File: ajax/libs/agility/package.json [hunk 1]
```diff
@@ -1,9 +1,15 @@
 {
   "name": "agility",
   "description": "Javascript MVC for the \"write less, do more\"",
-  "homepage": "https://github.com/arturadib/agility",
+  "homepage": "https://github.com/arturadib/agility.git",
```
Comment (pvnr0082t) [inline ajax/libs/agility/package.json]: @x09326 You don't need to change `homepage` in this commit.

**Comment by x09326** [comment]
@pvnr0082t 
Please help me review it again, thanks.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Comment by PeterDaveHello** [comment]
> "Modify homepage of agility" 

Why to modify? What's the detail of the modification? Should be much more clear and precise next time.

**Pull request merged**